src/rust/zh.manhuagui/src/decoder.rs

```diff
@@ -36,8 +36,7 @@ impl Decoder {
 			String::new()
 		};
 
-		let suffix_vec = vec![
-			self.tr(c % self.a, 36),
+		let suffix_vec = [self.tr(c % self.a, 36),
```

Comment:
why was the formatting here changed? it was better before

Reply:
clippy would complain about it when testing in my repository, so fix that.